tsMuxerGUI/translations/tsmuxergui_ru.ts

```diff
@@ -927,7 +927,7 @@
     <message>
         <location filename="../tsmuxerwindow.cpp" line="2506"/>
         <source>The output %1 &quot;%2&quot; already exists. Do you want to overwrite it?</source>
-        <translation>%1 &quot;%2&quot; уже есть. Хотите его перезаписать?</translation>
+        <translation>Файл %1 &quot;%2&quot; уже есть. Хотите его перезаписать?</translation>
```

Comment:
`%1` here is replaced with either "файл" or "каталог", depending on what the program needs to ask about - so this would result in something like "Файл каталог "foo" уже есть", which doesn't make much sense. ;-)

Reply:
Результат "foo" уже есть